### PR TITLE
Fix portfolios

### PR DESCRIPTION
**Pull request by jrapin**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/optimization/optimizerlib.py [hunk 1]
```diff
@@ -1060,7 +1061,7 @@ def intshare(n: int, m: int) -> Tuple[int, ...]:
             CMA(self.parametrization, num_workers=nw1),  # share parametrization and its rng
             TwoPointsDE(self.parametrization, num_workers=nw2),  # noqa: F405
             PSO(self.parametrization, num_workers=nw3),
-            SQP(self.parametrization, 1),  # noqa: F405
+            SQP(self.parametrization, num_workers=1),  # noqa: F405
```
Comment (jrapin) [inline nevergrad/optimization/optimizerlib.py:1064]: @teytaud FYI this is critical and can make a huge difference in ParaPorfolio and SQPCMA

Comment (teytaud) [reply nevergrad/optimization/optimizerlib.py:1064]: Waow!

**Comment by teytaud** [comment]
presumably the FAILED test is an artifact...

**Comment by jrapin** [comment]
> presumably the FAILED test is an artifact...

No, there is something weird happening in the CI, creating a segfault, I'll deactivate the test :s (it's not a very important one)

**Comment by jrapin** [comment]
The bugs seem unrelated, I'll merge this PR since it's a fix, and investigate further in another one

**Pull request merged**